src/sagemaker/workflow/pipeline.py

```diff
@@ -912,6 +947,25 @@ class _ExecutionList:
     pipeline_executions: Sequence[_PipelineExecution] = attr.ib(factory=list)
     next_token: str = attr.ib(default=None)
 
+    def table(self):
```

Comment:
This method seems to be duplicate with the _PipelineList's table, we should extract these common methods to a parent class - `_ListBase` class.

If don't have time, please add # TODO comments for the next step of improvements